Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -192,6 +199,177 @@ def download_export_scan(self, scan_id: str, file_id: str, file_format: str) ->
+
+    def download_vulnerabilities_chunk(self, export_uuid: str, chunk_id: int):
+        """
+
+        Args:
+            export_uuid: The UUID of the vulnerabilities export job.
+            chunk_id: The ID of the chunk you want to export.
+
+        Returns: Chunk of vulnerabilities from API.
+
+        """
+        return self._http_request(method='GET', url_suffix=f'/vulns/export/{export_uuid}/chunks/{chunk_id}',
+                                  headers=self._headers)
+
+    def export_assets_request(self, fetch_from):
+        """
+
+        Args:
+            chunk_size: maximum number of assets in one chunk.
+            fetch_from: the last asset that was fetched previously.
+
+        Returns: The UUID of the assets export job.
+
+        """
+        payload = {
+            'chunk_size': CHUNK_SIZE,
+            "filters": {
+                "updated_at": fetch_from
+            }
+        }
+        demisto.debug(f"my payload is: {payload}")
+        res = self._http_request(method='POST', url_suffix='assets/export', json_data=payload,
+                                 headers=self._headers)
+        return res.get('export_uuid')
+
+    def get_export_assets_status(self, export_uuid):
+        """
+        Args:
+                export_uuid: The UUID of the assets export job.
+
+        Returns: The assets' chunk id.
+
+        """
+        res = self._http_request(method='GET', url_suffix=f'assets/export/{export_uuid}/status', headers=self._headers)
+        return res.get('status'), res.get('chunks_available')
+
+    def download_assets_chunk(self, export_uuid: str, chunk_id: int):
+        """
+
+        Args:
+            export_uuid: The UUID of the assets export job.
+            chunk_id: The ID of the chunk you want to export.
+
+        Returns: Chunk of assets from API.
+
+        """
+        return self._http_request(method='GET', url_suffix=f'/assets/export/{export_uuid}/chunks/{chunk_id}',
+                                  headers=self._headers)
+
+    def get_export_uuid(self, num_assets: int, last_found: Optional[float], severity: List[str]):
+        """
+
+        Args:
+            num_assets: number of assets used to chunk the vulnerabilities.
+            last_found: vulnerabilities that were last found between the specified date (in Unix time) and now.
+            severity: severity of the vulnerabilities to include in the export.
+
+        Returns: The UUID of the vulnerabilities export job.
+
+        """
+        payload: dict[str, Any] = {
+            "filters":
+                {
+                    "severity": severity
+                },
+            "num_assets": num_assets
+        }
+        if last_found:
+            payload['filters'].update({"last_found": last_found})
+
+        res = self._http_request(method='POST', url_suffix='/vulns/export', headers=self._headers, json_data=payload)
+        return res.get('export_uuid', '')
+
+    def get_export_status(self, export_uuid: str):
```

Comment:
since we have something similiar to this for assets please change the name to 
get_vulns_export_status
or perhaps with the full word
get_vulnerabilities_export_status